R/readMultisep.R

```diff
@@ -1,8 +1,7 @@
 readMultisep <- function(file,sep) {
+  checkmate::assert_character(sep)
```

Comment:
please add len=1

Reply:
hmm, i suppose i should write a test for this function -- will put this on my todo

Reply:
but shall I merge or just leave this pull request open?

Reply:
yes please merge, i will do this separately